news/5889.bugfix

```diff
@@ -0,0 +1,6 @@
+Pip would like to ensure that requirements from pypi do not install dependencies from URLs.
```

Comment:
This should be a lot shorter and to the point. It should state the issue fixed from the user's perspective rather than referencing the code. For example, it could have the form, "Fixed crash when installing a requirement with such-and-such format..."

Reply:
Done.